forte/solvers/feature.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum, auto
+
+
+class Feature(Enum):
+    """
+    This enum class is used to store all types of features
```

Comment:
Document what a "feature" is. I think you are the person in this group most acquainted with computational graph technical jargon.